upload/lambdas/batch_watcher/batch_watcher.py

```diff
@@ -0,0 +1,139 @@
+import os
+import json
+
+import boto3
+import requests
+
+from upload.common.retry import retry_on_aws_too_many_requests
+from upload.common.logging import get_logger
+from upload.common.database import run_query, run_query_with_params
+from upload.common.exceptions import UploadException
+
+logger = get_logger(__name__)
+
+
+class BatchWatcher:
+    def __init__(self):
+        self.api_key = os.environ["INGEST_API_KEY"]
+        self.deployment_stage = os.environ["DEPLOYMENT_STAGE"]
+        self.batch_client = boto3.client("batch")
+        self.ec2_client = boto3.client('ec2')
+        self.lambda_client = boto3.client('lambda')
+        if self.deployment_stage == "prod":
+            self.upload_url = "https://upload.data.humancellatlas.org"
```

Comment:
Don't we pass API_HOST into these Lambdas?
This code hardwires the Upload Service to be only usable by the HCA.

Reply:
Good catch. Will fix